spec/support/pages/pages/sign_in_page.rb

```diff
@@ -0,0 +1,18 @@
+class SignInPage < SitePrism::Page
+  set_url'users/sign_in'
+  set_url_matcher %r{/users/sign_in}
+
+  # ELEMENTS
```

Comment:
That comment seems over-informative to me because I see three `elements` after the comment so I know that they are elements. Let's comment with code.